.github/workflows/test-v3.yml

```diff
@@ -29,4 +29,7 @@ jobs:
         hatch env create
     - name: Run Tests
       run: |
-        hatch run test:run
\ No newline at end of file
+        hatch run test:run
```

Comment:
It would be nice if we could use github action's matrix feature here as well. Obviously, we're skipping lots of tests at the moment so running the whole matrix in one action is fine. But as the test suite comes back to life, I think we'll spread out. It seems like Hatch can support this with something like: 

```
hatch run +py=${{ matrix.python-version }} test:run
```

_Note: I wasn't able to figure out how to run individual matrix members with hatch variable selection but it seems like it should be possible: https://hatch.pypa.io/1.9/cli/reference/#hatch-env-run_